network/src/noise/handshake.rs

```diff
@@ -248,15 +258,42 @@ impl NoiseUpgrader {
             .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?;
 
         // send the first handshake message
+        sl_info!(
+            network_log(network_events::NOISE_UPGRADE, &self.network_context)
+                .data("origin", "client")
+                .data("step", "writing_data")
+                .log(format!(
+                    "{} noise client writing_data: {}",
+                    self.network_context, remote_public_key
```

Comment:
why not client_message?

Reply:
I imagine neither are particularly parseable but at remote_public_key is likely more so.

Reply:
my thinking was: if we want to use this for debugging we might want to print out the actual message we send, not just part of it

Reply:
sure, but the entirety of the data is actually in this log line... if we want to go the route of the actual message, we should make these proper lcs serializable structs that can then be pretty printed.